streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -446,6 +446,13 @@ void print(final Serde<K> keySerde,
      * @param printed options for printing
      */
     void print(final Printed<K, V> printed);
+    
+    /**
+     * Merges the given streams into one larger stream. 
+     * @param a stream which is  to be merged.
+     * No guarantee of order
+     */
+    KStream<K, V> merge(KStream<K, V> stream);
```

Comment:
Nit: add `final` -- we add `final` where ever possible. (ie. `merge(final KStream....`)